students/Sahlberg/Lesson2/Assignment2_Functions.py

```diff
@@ -0,0 +1,52 @@
+"""Ian Sahlberg
+Assignment 2 - Grid functions
+Self paced
+December 10, 2018"""
+
+def func1():
+    """This function prints out a simple box."""
+    print("+ - - - - + - - - - +\n|         |         |\n|         |         |\n|         |         |\n|         |         |")
+    print("+ - - - - + - - - - +\n|         |         |\n|         |         |\n|         |         |\n|         |         |")
+    print("+ - - - - + - - - - +")
```

Comment:
just a suggestion for function names:
`print_grid_simple`
`print_grid_by_size`
`print_grid_by_size_width`

always best to be clear and descriptive with your functions.